src/en/readcomic/src/eu/kanade/tachiyomi/extension/en/readcomic/Readcomic.kt

```diff
@@ -0,0 +1,199 @@
+package eu.kanade.tachiyomi.extension.en.readcomic
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Readcomic : ParsedHttpSource() {
+
+    override val name = "ReadComic"
```

Comment:
also change name and ext name/package to "ReadComic.net" or similar, since these comic sites have very similar names